marimo/_plugins/validators.py

```diff
@@ -57,3 +59,21 @@ def warn_js_safe_number(*values: int | float | None) -> None:
                 "integers in JavaScript. This may cause precision issues.",
                 stacklevel=3,
             )
+
+
+def validate_no_integer_columns(df: Any) -> None:
+    if not DependencyManager.pandas.has():
+        return
```

Comment:
For performance this should be `not DependencyManager.pandas.imported()`, since `df` can't be a Pandas dataframe if Pandas is not imported. Importing pandas can take a long time.